packages/client/hmi-client/src/page/project/tera-project.vue

```diff
@@ -80,6 +80,7 @@
 				<tera-stratify
 					v-if="workflowNode && workflowNode.operationType === WorkflowOperationTypes.STRATIFY"
 					:node="workflowNode"
+					:key="workflowNode.id"
```

Comment:
Maybe kind of a "cheat", but this is to make the tera-stratify component fully re-render everytime a different stratify drilldown is opened, because how Vue's internal logic handles reactively updating a component causes a lot of difficult to handle edge cases.